addons/mail/static/src/core/common/record.js

```diff
@@ -105,7 +105,7 @@ export class Record {
     Model;
 
     delete() {
-        delete this.Model.records[this.localId];
+        delete this.Model?.records[this.localId];
```

Comment:
Already fixed by https://github.com/odoo/odoo/pull/134254
Better remove from diff, so it doesn't conflict